src/main/java/frc/trigon/robot/subsystems/swerve/Swerve.java

```diff
@@ -88,54 +87,57 @@ public Rotation2d getHeading() {
         return Rotation2d.fromDegrees(SwerveConstants.GYRO.getSignal(Pigeon2Signal.YAW));
     }
 
-    public Translation3d getFieldRelativeVelocity3d() {
-        return new Translation3d(
-                getFieldRelativeVelocity().vxMetersPerSecond,
-                getFieldRelativeVelocity().vyMetersPerSecond,
-                0
-        );
+    public ChassisSpeeds getFieldRelativeChassisSpeeds() {
+        final ChassisSpeeds selfRelativeSpeeds = getSelfRelativeChassisSpeeds();
+        return ChassisSpeeds.fromRobotRelativeSpeeds(selfRelativeSpeeds, RobotContainer.ROBOT_POSE_ESTIMATOR.getEstimatedRobotPose().getRotation());
     }
 
-    public ChassisSpeeds getFieldRelativeVelocity() {
-        final ChassisSpeeds selfRelativeSpeeds = getSelfRelativeVelocity();
-        return ChassisSpeeds.fromRobotRelativeSpeeds(selfRelativeSpeeds, RobotContainer.ROBOT_POSE_ESTIMATOR.getEstimatedRobotPose().getRotation());
+    public Transform2d getFieldRelativeVelocity() {
```

Comment:
This is confusing to say the least
Transforms don't represent velocity, they represent transformations. What should this transform be applied on? A self relative transform makes more sense because of how transforms work.